components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt

```diff
@@ -284,6 +291,20 @@ internal class DisplayToolbar(
         invalidateActions(navigationActions + pageActions + browserActions)
     }
 
+    /**
+     * Set a LongClickListner to the urlView of the toolbar.
+     */
+    private fun setOnUrlLongClickListener(handler: (() -> Unit)? = null) {
+        if (handler == null) {
+            urlView.setOnLongClickListener(null)
```

Comment:
Do we need to set the click listener to null? Can we not just do the else case always? 🤔 

Maybe we could even make the listener non-null as well.

Reply:
@jonalmeida We probably could. While writing this I tested and the longpress always has the haptic feedback, so I thought there might be a case someone wants to fully remove it.

Reply:
Oh, interesting. I didn't realize the system hooks into that and does that when a listener is attached.